### DISCUSSION TITLE
params.hpp reset after server-start

### DISCUSSION THREAD

**Discussion by stroozey** [official-antistasi-community/A3-Antistasi#2341] [Q&A] [answered]
I am hosting a LAN server and playing with a couple of friends. As the game seems rather difficult if you are playing with 2-3 people, I wanted to change the params.hpp. Unfortunately whenever I start the server after an edit in the file, the file keeps resetting even though the params where changed and saved in the file beforehand.

**Comment by Lazejun** [comment, marked as answer]
Well if you edit a Workshop file it will just Update to what it's supposed to be.
Don't edit the Files from the Workshop.

There is no point to Editing Params.hpp Manually, Parameters save for Multiple Versions already, there are 3 How to Hints at the Top of the Parameters explaining how they work.

**Reply by stroozey** [reply]
Thank you so much for the fast reply. I really appreciate your work!

**Comment by Bob-Murphy** [comment]
Laze was faster but I still finish my text.

Hi, I guess you are editing the file that you got through the steam workshop when subscribing to our item. That doesn't work.
As soon as you change something in the steam files steam will think that the file is broken and repair it.
When starting a campaign you can simply change the parameters on the ingame parameters tab which can be foung top right on the role selection screen.
When you then start the game and save once ingame on the white board, the selected parameters will be saved in the vars file aka the game save.
On the next start the parameters will again SHOW the standards but the changes you selected beforehand WILL be applied once the mission is loaded.
This is a technical limitation we can not change and you have to the save system.

If you still want to manually change the params.hpp you have to download the files from the GitHub release page, unpack the pbo, make the changes, save them and then use that files with for example putting them in the C:\Users\username\Documents\Arma 3 - Other Profiles\playername\mpmissions folder. 
But that is normally not necessary as the ingame system works.

**Reply by stroozey** [reply]
Thank you so much for the detailed instructions.
